packages/composites/savax-price/src/endpoint/price.ts

```diff
@@ -0,0 +1,60 @@
+import { AdapterContext, AdapterRequest, AdapterResponse } from '@chainlink/types'
+import { Config, FLOATING_POINT_DECIMALS } from '../config'
+import * as TA from '@chainlink/token-allocation-adapter'
+import { ethers } from 'ethers'
+
+export const supportedEndpoints = ['price']
+
```

Comment:
@austinborn do we still need to export input parameters even though the EA doesn't have any?

Reply:
No, although the README script will need to be updated to support no `inputParameters` when we want to start generating READMEs for composite adapters. I'll make a note for it in the relevant ticket.